### ISSUE TITLE
Plotting points showing estimated detection prob appears wrong for point transect analyses with adjustments

### ISSUE THREAD

**Issue by lenthomas** [DistanceDevelopment/Distance#134]
In the following analysis there are no covariates, so I would expect the points to sit exactly on the detection function line.  Therefore either the line or the points are wrong.  The same thing happens for other key functions if there are adjustment terms.

```{r}
DuikerCameraTraps <- read.csv(file="https://datadryad.org/stash/downloads/file_stream/73221", 
                              header=TRUE, sep="\t")

library(Distance)
conversion <- convert_units("meter", NULL, "square kilometer")
trunc.list <- list(left=2, right=15)
mybreaks <- c(seq(2, 8, 1), 10, 12, 15)

uni1 <- ds(DuikerCameraTraps, transect = "point", key="unif", adjustment = "cos",
           nadj=1,
           cutpoints = mybreaks, truncation = trunc.list)

plot(uni1, showpoints = TRUE)
```
<img width="859" alt="Screenshot 2022-10-09 213116" src="https://user-images.githubusercontent.com/129519/194778192-266be505-f00d-4d4d-85f4-a49195c7094f.png">

**Comment by dill** [comment]
Looking at other datasets, this seems to be an issue with left truncation, rather than just point transects with adjustments.

**Comment by dill** [comment]
closed via [451badf](https://github.com/DistanceDevelopment/mrds/commit/451badf469902dfa6f1b76ba713973b8516919fe)
